### PR TITLE
Fix argument disable_dynamic_ip

### PR DESCRIPTION
**Pull request by ChoiZ**
"disable_dynamic_ip": [REMOVED] Removed in favor of enable_dynamic_ip.

### CODE REVIEW

File: website/docs/r/instance_server.html.markdown [hunk 1]
```diff
@@ -57,7 +57,7 @@ resource "scaleway_instance_server" "web" {
   tags = [ "hello", "public" ]
 
   ip_id = scaleway_instance_ip.ip.id
-  disable_dynamic_ip = true # required for now when using `ip_id`
```
Comment (kindermoumoute) [inline website/docs/r/instance_server.html.markdown]: The whole line can be removed as `enable_dynamic_ip = false` is the default value for this attribute

File: website/docs/r/instance_server.html.markdown [hunk 2]
```diff
@@ -57,7 +57,7 @@ resource "scaleway_instance_server" "web" {
...
+  enable_dynamic_ip = false # required for now when using `ip_id`
```
Comment (kindermoumoute) [inline website/docs/r/instance_server.html.markdown]:
```suggestion
```

File: website/docs/r/instance_server.html.markdown (hunk 1 above)
Comment (ChoiZ) [reply website/docs/r/instance_server.html.markdown]: Oh yes of course :)

**Comment by QuentinBrosse** [comment]
Fix #415

**Pull request merged**